ocs_ci/utility/utils.py

```
@@ -1418,37 +1523,155 @@ def get_rosa_cli(
+                        if binary_path_exists:
+                            backup_file = f"{binary_path}.bak"
+                            os.rename(binary_path, backup_file)
+                            try:
+                                shutil.move(temp_binary, binary_path)
+                                delete_file(backup_file)
+                                log.info("Deleted backup binaries.")
+                            except FileNotFoundError as ex:
+                                log.error(
+                                    "Something went wrong with copying binary, reverting backup file. "
+                                    f"Exception: {ex}"
+                                )
```

Ruff (0.15.15)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)